src/data/finset/basic.lean

```diff
@@ -402,6 +402,9 @@ nonempty_iff_ne_empty.not.trans not_not
 theorem eq_empty_or_nonempty (s : finset α) : s = ∅ ∨ s.nonempty :=
 classical.by_cases or.inl (λ h, or.inr (nonempty_of_ne_empty h))
 
+lemma nonempty_of_mem {a : α} {s : finset α} (h : a ∈ s) : s.nonempty :=
+nonempty_of_ne_empty (ne_empty_of_mem h)
```

Comment:
```suggestion
⟨_, h⟩
```
... or maybe you don't actually need this lemma

Reply:
Oops. Looks like I don't need it indeed.